x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/sub_action_framework/index.ts

```diff
@@ -7,41 +7,87 @@
 
 import type SuperTest from 'supertest';
 import expect from '@kbn/expect';
+import {
+  SENTINELONE_CONNECTOR_ID,
+  SUB_ACTION,
+} from '@kbn/stack-connectors-plugin/common/sentinelone/constants';
+import { Role } from '@kbn/security-plugin/common';
+import { PRIVILEGE_ID } from '@kbn/security-solution-features/src/security/kibana_sub_features';
+import { ToolingLog } from '@kbn/tooling-log';
 import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
 import { getUrlPrefix, ObjectRemover } from '../../../../../common/lib';
 
+let log: ToolingLog;
+const DEFAULT_SECRETS = Object.freeze({ username: 'elastic', password: 'changeme' });
+
+interface LogErrorDetailsInterface {
+  (this: SuperTest.Test, err: Error & { response?: any }): SuperTest.Test;
+  ignoreCodes: (
+    codes: number[]
+  ) => (this: SuperTest.Test, err: Error & { response?: SuperTest.Response }) => SuperTest.Test;
+}
+
+const logErrorDetails: LogErrorDetailsInterface = function (err) {
```

Comment:
Just a helper function for use with `supertest` that will log the API response body when errors occur (assist with debugging)